packages/bot-web-ui/src/components/dashboard/dashboard-component/load-bot-preview/recent-workspace.tsx

```diff
@@ -5,45 +5,31 @@ import { isDesktop, isMobile } from '@deriv/shared';
 import classnames from 'classnames';
 import { DBOT_TABS } from 'Constants/bot-contents';
 import React from 'react';
-import { connect } from 'Stores/connect';
-import RootStore from 'Stores/index';
 import { useComponentVisibility } from '../../hooks/useComponentVisibility';
 import { CONTEXT_MENU_MOBILE, MENU_DESKTOP, STRATEGY } from './constants';
 import './index.scss';
+import { useDBotStore } from 'Stores/useDBotStore';
 
 type TRecentWorkspace = {
-    active_tab: number;
-    dashboard_strategies: [];
-    getRecentFileIcon: (string: string) => void;
-    getSaveType: (type: string) => string;
     index: number;
-    getSelectedStrategyID: (current_workspace_id: string) => void;
-    loadFileFromRecent: () => void;
-    onToggleDeleteDialog: (is_delete_modal_open: boolean) => void;
-    previewRecentStrategy: (workspaceId: string) => void;
-    selected_strategy_id: string;
-    setActiveTab: (active_tab: number) => void;
-    setPreviewOnDialog: (has_mobile_preview_loaded: boolean) => void;
-    toggleSaveModal: () => void;
     workspace: { [key: string]: string };
 };
 
-const RecentWorkspace = ({
-    active_tab,
-    dashboard_strategies,
-    getRecentFileIcon,
-    getSaveType,
-    index,
-    loadFileFromRecent,
-    onToggleDeleteDialog,
-    previewRecentStrategy,
-    selected_strategy_id,
-    getSelectedStrategyID,
-    setActiveTab,
-    setPreviewOnDialog,
-    toggleSaveModal,
-    workspace,
-}: TRecentWorkspace) => {
+const RecentWorkspace = ({ workspace, index }: TRecentWorkspace) => {
```

Comment:
@rupato-deriv Make sure to wrap the components with `observer`, Otherwise rerender won't happen.

Reply:
done ✅